src/components/portrait/portrait-test.stories.tsx

```diff
@@ -50,3 +50,18 @@ Default.args = {
   size: "M",
   shape: "square",
 };
+
+export const PortraitDefaultComponent = ({ ...props }) => {
+  return <Portrait {...props} />;
+};
+
+export const PortraitComponent = ({ ...props }) => {
+  return (
+    <Portrait
+      tooltipMessage="Rebecca Smith"
+      tooltipIsVisible
+      src="https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light"
+      {...props}
+    />
+  );
+};
```

Comment:
Just a question on line 54-67, is it necessary to create both PortraitDefaultComponent and PortraitComponent? @DlgSHi